RELEASENOTES-1.4.docu

```diff
@@ -558,4 +558,17 @@
       in multiple inheritance: it allows any method that overrides multiple
       implementations from unrelated templates to individually reference and
       call the implementations it overrides.</add-note></build-id>
+  <build-id _6="next" _7="next"><add-note>Added a system
+      for <em>provisional language features</em>: language features
+      that are disabled by default but can be explicitly enabled with
+      a special syntax. The first provisional feature
+      is <tt>explicit_param_decls</tt>, which can be enabled per file
```

Comment:
You should split these up to separate releasenotes. One for the provisional system, one for the `explicit_param_decls`. By the way, don't provisionals have a JIRA of their own, or did we never create one?

Reply:
I didn't find a jira from a quick search.

I have split the releasenote.

Reply:
If you have, you haven't pushed that up!